src/main/java/com/basho/riak/client/core/query/timeseries/CoverageEntry.java

```diff
@@ -0,0 +1,151 @@
+package com.basho.riak.client.core.query.timeseries;
+
+import java.io.Serializable;
+import java.util.Arrays;
+
+public class CoverageEntry implements Serializable {
```

Comment:
Are these Coverage* classes exclusive to timeseries queries, or can we move them to a higher level package?